source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -90,6 +90,11 @@ type State = {
   isTransactionFeeCalculated: boolean,
 };
 
+const getTransactionFeeCalculated = (
```

Comment:
@danielmain please rename to `getTransactionTotalAmount` as the current name is a bit confusing...